src/function.ts

```diff
@@ -140,6 +275,78 @@ abstract class FunctionBase<P, O>
       ResultSelector: "$.Payload",
     };
   }
+
+  protected static normalizeAsyncDestination<P, O>(
```

Comment:
Why protected?

Reply:
This is an abstract base class, this method is used by the base and child class. And we don't need to expose this.